### PR TITLE
[auto] pytorch-pr-6967

### PR DESCRIPTION
**Pull request by onnxbot-worker-1**
This is an auto PR managed by onnxbot for tracking

https://github.com/pytorch/pytorch/pull/6967 Fix the symbolic of batchnorm to handle special case

### CODE REVIEW

**Comment by houseroad** [comment]
@onnxbot retest this please
